### PR TITLE
Verbiage change: Annotation Types -> Attributes

### PR DESCRIPTION
**Pull request by kadamwhite**
Also renames `.demographics` store property to `.demographicAttributes`,
for clarity: updated store structure is,

```
demographicAttributes: {
  byId: {},
  order: [],
}
```
The store could probably be restructured more in the future once we have other types of attributes but I wanted to keep only one level of subreducers within each top-level property, for now

### CODE REVIEW

File: frontend/components/PerceivedDemographics.jsx [hunk 1]
```diff
@@ -44,13 +44,13 @@ class PerceivedDemographics extends Component {
 
   // Prepare the annotations into the format expected by the action/reducer
   prepareAnnotationsObject() {
-    const demographics = this.props.questionOrder.map(questionName => ({
+    const annotations = this.props.questionOrder.map(questionName => ({
       name: questionName,
-      option: this.state[questionName],
+      value: this.state[questionName],
```
Comment (kadamwhite) [inline frontend/components/PerceivedDemographics.jsx]: This is actually what the server was expecting, so I think I was passing the wrong values up until now

**Comment by kadamwhite** [comment]
Got a go-ahead in slack!

**Comment by boazsender** [comment]
+1

@kadamwhite can you open an issue to to refactor the backend and include the api you're expecting?

**Comment by kadamwhite** [comment]
@boazsender Sure, will do

**Pull request merged**